src/features/feeds/shared/using-price-feeds.mdx

```diff
@@ -24,18 +29,20 @@ The contract has the following components:
 
 ## Compile, deploy, and run the contract
 
-:::caution[ Configure and fund your wallet]
+<Aside type="caution" title="Configure and fund your wallet">
 
 If you have not already configured your MetaMask wallet and funded it with testnet ETH, follow the instructions in [Deploy Your First Smart Contract](/getting-started/deploy-your-first-contract#install-and-fund-your-metamask-wallet) to set that up. You can get testnet ETH at one of the available [Sepolia faucets](/resources/link-token-contracts/#sepolia-testnet).
 
-:::
+</Aside>
 
 Deploy the `PriceConsumerV3` smart contract on the Sepolia testnet.
 
 1. [Open the example contract](https://remix.ethereum.org/#url=https://docs.chain.link/samples/PriceFeeds/PriceConsumerV3.sol) in Remix. Remix opens and shows the contents of the smart contract.
 
    <div class="remix-callout">
-   <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/PriceFeeds/PriceConsumerV3.sol">Open the contract in Remix</a>
+     <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/PriceFeeds/PriceConsumerV3.sol">
+       Open the contract in Remix
+     </a>
```

Comment:
```suggestion
     <a href="https://remix.ethereum.org/#url=https://docs.chain.link/samples/PriceFeeds/PriceConsumerV3.sol" target="_blank">
       Open the contract in Remix
     </a>
```

This one has the CSS issue again. I think you said `target="_blank"` fixes it?

Reply:
I thought _removing_ the `target` attribute fixed this, but now I think it's actually forcing the link to go within one line in the source code. https://github.com/smartcontractkit/documentation/pull/1206/commits/be8cb3a136ffd6cb75a9a4167b624e3f223246db

The issue doesn't make sense to me in the first place, so I don't know why this fixes it, but it seems to work.